cads_adaptors/adaptors/roocs/__init__.py

```diff
@@ -11,28 +12,27 @@ def __init__(self, *args, **kwargs):
         self.facets = self.config.get("facets", dict())
         self.facet_groups = self.config.get("facet_groups", dict())
         self.facets_order = self.config.get("facets_order", [])
+        self.facet_search = self.config.get("facet_search", dict())
+        self.operators = self.config.get("operators", dict())
 
     def retrieve(self, request: Request) -> BinaryIO:
         from cads_adaptors.tools import download_tools, url_tools
 
         os.environ["ROOK_URL"] = "http://rook.dkrz.de/wps"
```

Comment:
Define from adaptor config like ROOK_MODE
```suggestion
        os.environ["ROOK_URL"] = self.config.get("ROOK_URL","http://rook.dkrz.de/wps")
```